src/test/java/com/sugarcrm/candybean/webservices/WSSystemTest.java

```diff
@@ -76,15 +73,17 @@ public void testHTTPError() {
 	}
 
 	@Test
+	@Ignore("This test should pass, but takes a full minute to do so because it" +
+			"waits for the response to time out.")

```

Comment:
Updating to the latest httpclient made me realize that I wasn't testing what I though I was testing. The "Target host is null" error comes from having a malformed URL. Correcting the "correct" url causes httpclient to timeout. The test new tests the correct thing and passes, however it take a minute to timeout, so I ignored the test.